src/zsys.c

```diff
@@ -1392,17 +1392,16 @@ zsys_set_logsender (const char *endpoint)
     if (endpoint) {
         //  Create log sender if needed
         if (!s_logsender) {
-            s_logsender = zsys_socket (ZMQ_PUB, NULL, 0);
+            s_logsender = zsock_new_pub(NULL);
```

Comment:
I do not understand your reference to "here".

If you mean zsock_attach() is not needed - it is, if you want to use the ">"/"@" connect/bind syntax for the endpoint which zsys_socket() does not support.